### ISSUE TITLE
Thoughts on running models other than Llama in the android app

### ISSUE THREAD

**Issue by j0h0k0i0m** [pytorch/executorch#6804]
As the title suggests, is it possible to run models other than LLaMA in the app? 
In particular, I'd like to try running Phi 3.5 mini models in the application. 
Is there a way to do this, or are there any plans to support it in the future?

**Comment by larryliu0820** [comment]
Yeah we have a plan to expose `extension/llm` export APIs so that other LLMs (such as Phi 3.5 mini) can leverage them and generate the .pte file that is runnable on demo apps. Please find this RFC for more details: https://github.com/pytorch/executorch/issues/6558. We will work on this in the coming weeks.

With the .pte files generated, along with the tokenizer model, the demo app will be able to run the model without issue.
